### DISCUSSION TITLE
connection string

### DISCUSSION THREAD

**Discussion by Fehard22** [balta-io/forum#153] [Tira Dúvidas] [answered]
quando tento executar o comando da string connection, o terminar apresenta o erro conforme a imagem, acredito que por essa razão, não aparece nenhuma outra database além da Default.

![errosql](https://user-images.githubusercontent.com/119022383/206820487-bb364da3-4da7-4460-b167-4a762ba70dcb.png)

**Comment by andrebaltieri** [comment, marked as answer]
@Fehard22 esta linha não é um comando! Ela deve ficar no arquivo `appsettings.json` conforme mostrado nos cursos.
Por isto está recebendo o erro.
